### ISSUE TITLE
[java] Disable LoosePackageCoupling by default

### ISSUE THREAD

**Issue by mcandre** [pmd/pmd#1600]
**Affects PMD Version:** 

5.0+

**Rule:**

LoosePackageCoupling

**Description:**

This rule requires explicit configuration for use. Let's either supply the rule with sufficient default configuration to actually run, or turn it off by default in the standard Java rule set.

As a workaround, I am configuring this in my rulesets.xml:

```xml
<rule ref="category/java/design.xml">
    <exclude name="LoosePackageCoupling" />
</rule>
```

**Comment by jsotuyod** [comment]
> or turn it off by default in the standard Java rule set.

Actually, there is no *"default standard ruleset"*. The categories are there to define rules in an orderly way, grouped together in a way that makes sense. They *can* be included as a whole for ease of use (ie: not having to list each rule individually), but users are always encouraged to [write their own rulesets](https://pmd.github.io/pmd-6.10.0/pmd_userdocs_making_rulesets.html) configuring the rules as it best fits their needs.

PMD doesn't intend to be a one-size-fits-all set of rules everyone should live by; but a toolbox teams can use to enforce given code styles and best practices.

The only non-deprecated user-ready ruleset shipped with PMD, which is the [java-quickstart](https://github.com/pmd/pmd/blob/master/pmd-java/src/main/resources/rulesets/java/quickstart.xml) ruleset; intended as a good starting point for new users, doesn't include that rule by default.

**Comment by mcandre** [comment]
> Actually, there is no "default standard ruleset".

I was not aware of that. Perhaps there could be better coordination between PMD proper and the PMD Gradle plugin, as that plugin has some different behaviors and outdated data compared to the main project.

**Comment by adangel** [comment]
That's something we should consider: Improving the documentation in this area.

Gradle as well as maven provide a "default" ruleset - but if you are not aware, that you can change this default, then you run into problems as you describe (using rules that don't fit your need, not seeing rules, that might be useful for you/your project).

Would you mind helping us to improve the documentation?

For maven, we have e.g. this page: https://pmd.github.io/pmd-6.12.0/pmd_userdocs_tools_maven.html 
We lack however a dedicated documentation on how to use PMD with gradle.
